src/dos/program_imgmount.cpp

```diff
@@ -342,7 +340,7 @@ void IMGMOUNT::Run(void)
 				        drive_letter(dummy));
 			}
 		}
-		if (S_ISDIR(test.st_mode)) {
+		if (is_directory(temp_line)) {
 			WriteOut(MSG_Get("PROGRAM_IMGMOUNT_MOUNT"));
 			return;
 		}
```

Comment:
Looks like you fixed a bug here too.  Before, if that first `stat` call failed, the value of `struct stat test` is possibly undefined (we didn't zero clear it).